### PR TITLE
feat: show secret status on deployment screen

### PR DESCRIPTION
**Pull request by robot9706**
Show a status for each secret on the deployment page:
- Gray: If the container does not exist.
- Red: If the secret is missing.
- Green: If the secret is deployed.

### CODE REVIEW

File: protobuf/proto/common.proto [hunk 1]
```diff
@@ -238,6 +238,8 @@ message SecretList { repeated UniqueKeySecretValue data = 1000; }
 
 message ListSecretsResponse {
   string prefix = 1;
-  string publicKey = 2;
-  repeated string keys = 3;
+  string name = 2;
+  string publicKey = 3;
```
Comment (robot9706) [inline protobuf/proto/common.proto:242]: Not sure why this was originally here :D Is this still required? @nandor-magyar

Comment (nandor-magyar) [reply protobuf/proto/common.proto:242]: The intent was to store the public key alongside the secret, we can decide if it is still valid or the node has a new public key.

**Review by balanceee — APPROVED** [review]
+1

File: Makefile [hunk 2]
```diff
@@ -29,7 +29,7 @@ proto-agent:
 # Generate API grpc files
 .PHONY: proto-crux
 proto-crux:
-	MSYS_NO_PATHCONV=1 docker run -u ${UID}:${GID} -v "${PWD}":/usr/work ghcr.io/dyrector-io/dyrectorio/alpine-proto:3.16 ash -c "\
+	MSYS_NO_PATHCONV=1 docker run --rm -u ${UID}:${GID} -v "${PWD}":/usr/work ghcr.io/dyrector-io/dyrectorio/alpine-proto:3.16 ash -c "\
```
Comment (m8vago) [inline Makefile:32]: whyy?

Comment (robot9706) [reply Makefile:32]: because it creates a new container every time? :/

**Review by nandor-magyar — APPROVED** [review]
Good job, sir! :tophat: 
G2G, after conflicts are resolved.

**Comment by robot9706** [comment]
Merging 🧩🧩🧩

**Pull request merged**